### PR TITLE
COM-1462 add test to check if when we duplicate a published activity,…

### PR DESCRIPTION
**Pull request by cmolle**
- [ ] Mon code est testé unitairement -np
- [x] Mon code est testé avec des tests d'intégration

- Périmetre interface : np

- Périmetre roles : np

- Cas d'usage : modification du test d'integration de duplication d'activite pour verifier que le status de la nouvelle activité est bien 'draft'.

### CODE REVIEW

File: tests/integration/steps.test.js [hunk 1]
```diff
@@ -194,19 +194,19 @@ describe('STEPS ROUTES - POST /steps/{_id}/activity', () => {
     });
 
     it('should duplicate an activity', async () => {
-      const duplicatedActivityId = activitiesList[0]._id;
```
Comment (sophiemoustard) [inline tests/integration/steps.test.js:197]: si jamais on enlève la status PUBLISHED, le test passe quand même.
tu pourrais du coup créer un autre test spécifique au status publié qui vérifie que l'activité de départ est publiée et celle créée ne l'est pas
et pour ce test tu gardes une activité non publiée

**Review by sophiemoustard — CHANGES_REQUESTED** [review]

**Review by KennyCallegari — APPROVED** [review]

**Review by KennyCallegari — APPROVED** [review]

File: tests/integration/steps.test.js [hunk 2]
```diff
@@ -232,6 +232,30 @@ describe('STEPS ROUTES - POST /steps/{_id}/activity', () => {
...
+          path: 'activities',
+          select: '-__v -createdAt -updatedAt',
+          populate: { path: 'cards', select: '-__v -createdAt -updatedAt' },
+        })
+        .lean();
+
+      const duplicatedActivity = await Activity.findById(duplicatedActivityId).lean();
+
+      expect(response.statusCode).toBe(200);
+      expect(stepUpdated.activities[0].status).toBe('draft');
```
Comment (CorentinMartinot) [inline tests/integration/steps.test.js:255]: je crois que tu verifies la mauvaise activité ( si on console.log c'est "gater le coin")

Comment (cmolle) [reply tests/integration/steps.test.js]: Bien vu ! C'est corrigé :)

**Review by sophiemoustard — APPROVED** [review]
